### ISSUE TITLE
Peer dependency warning with @slack/bolt v3 and npm 7.

### ISSUE THREAD

**Issue by mattcasey** [IBM/slack-wrench#99]
**Is your feature request related to a problem? Please describe.**
When using npm 7, I am forced to use the `--legacy-peer-deps` flag due to this library.

**Describe the solution you'd like**
Increase the versions allowed for @slack/bolt to <4.0.0. I can make a PR but wanted to see if that's OK before I invest the time.

**Describe alternatives you've considered**
Fork it myself

**Additional context**
I am using this library with version 3.3.0. This is the error I get when installing/uninstalling _any package_ in my project. I blame npm too because it shouldn't yell at me when I'm working with other packages, but would like to address the issue here:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: @northshore/process@1.0.0
npm ERR! Found: @slack/bolt@3.3.0
npm ERR! node_modules/@slack/bolt
npm ERR!   @slack/bolt@"^3.3.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @slack/bolt@">=1.0.0 <3.0.0" from @slack-wrench/koa-bolt@1.2.0
npm ERR! node_modules/@slack-wrench/koa-bolt
npm ERR!   @slack-wrench/koa-bolt@"^1.2.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Comment by barlock** [comment]
Thanks for opening the issue and volunteering to make the pr. 

If they're compatible, then yeah absolutely lets bump those peer dependencies. 

I need to take another look through the boltv4 changelog to see what might break. A colleague of mine found a non-obvious one that I helped with but I don't remember what exactly it was. I would imagine that bumping both the peer dep and dev dependencies so the tests run should catch most if not all issues if they show up though. So, bump the dev dep too.

**Comment by mattcasey** [comment]
OK, will do. To be clear, I'm actually just on v3.3 which is what's in master. So when we bump it to <4, and till be anything less than 4. Here's the changelog for bolt: https://github.com/slackapi/bolt-js/releases. There may be some stuff in there to improve how this works with koa, actually.

**Comment by barlock** [comment]
🤦 Sorry, I had friday brain. I haven't tested this with bolt 3, but it should be, and that's what I meant instead of `boltv4`

Yeah I do need to dig through what the new `HTTPReceiver` does and see if we can remove the `ExpressReceiver` as part of the implementation. Seems promising.

**Comment by barlock** [comment]
Fix published in `1.3.0` https://www.npmjs.com/package/@slack-wrench/koa-bolt
